### PR TITLE
Switch to env variables

### PR DESCRIPTION
**Pull request by tamas-molnar**

### CODE REVIEW

**Comment by george-angel** [comment]
1. Do you still need `ADD config.json.template /v1-suggestor/config.json`? File is no longer there, I'm guessing that has been moved to the app
2. Trying to build:

```
# github.com/Financial-Times/v1-suggestor
./logging.go:12: undefined: log.LUTC
./logging.go:12: const initializer log.Ldate | log.LUTC is not a constant
```

because golang version in alpine is 1.4:

```
/ # apk info go
go-1.4.2-r1 description:
Go programming language compiler

go-1.4.2-r1 webpage:
http://www.golang.org/

go-1.4.2-r1 installed size:
128991232
```

File: Dockerfile [hunk 1]
```diff
@@ -16,9 +16,8 @@ RUN apk add --update bash \
   && go test ./... \
   && go build \
   && mv v1-suggestor /app \
-  && mv config.json /config.json \
   && apk del go git bzr \
   && rm -rf $GOPATH /var/cache/apk/*
 
 ENTRYPOINT [ "/bin/sh", "-c" ]
```
Comment (george-angel) [inline Dockerfile]: Not needed, this is already the default ENTRYPOINT

File: Dockerfile [hunk 2]
```diff
@@ -16,9 +16,8 @@ RUN apk add --update bash \
...
-CMD [ "/startup.sh" ] 
+CMD [ "./app" ]
```
Comment (george-angel) [inline Dockerfile]: Very small detail, but the `.` isn't needed, go compiler spits out an executable binary

**Comment by tamas-molnar** [comment]
Alpine 3.3 has go 1.5, I updated the imagefile to pull that down. I thought latest was the default for the version and it will pull that down.

**Comment by george-angel** [comment]
True enough, my docker daemon just used the existing version (3.2)

**Comment by george-angel** [comment]
+1

**Pull request merged**